### PR TITLE
Send notifications with Svelte and Novu Guide

### PR DESCRIPTION
**Pull request by unicodeveloper**

### CODE REVIEW

File: echo/integrations/svelte-email.mdx [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+title: "Svelte E-mail"
+---
+
+Integrating Novu Echo with [Svelte email](https://react.email/) for your Svelte application can be done in three steps. If you don't have an app, you can [clone our Svelte example](https://github.com/novuhq/novu-svelte-email).
+
+<Steps>
+  <Step title="Install Svelte email components">
+    Install the required Svelte email components.
```
Comment (sumitsaurabh927) [inline echo/integrations/svelte-email.mdx:9]: Is it one component or many?

**Review by sumitsaurabh927 — CHANGES_REQUESTED** [review]

File: echo/integrations/svelte-email.mdx (hunk 1 above)
Comment (unicodeveloper) [reply echo/integrations/svelte-email.mdx:9]: Many components in one package. @sumitsaurabh927

Comment (sumitsaurabh927) [reply echo/integrations/svelte-email.mdx:9]: @unicodeveloper 
Oh, okay. 
So is it correct to write 'email components' or shall we write 'email component'?

Comment (unicodeveloper) [reply echo/integrations/svelte-email.mdx:9]: It's correct to write 'email components'. That's how we wrote it for the others. Another alternative is to just use "package" but I want it to be consistent with the other integration pages. @sumitsaurabh927

**Review by sumitsaurabh927 — APPROVED** [review]

**Pull request merged**